### ISSUE TITLE
Collecting dropped C4

### ISSUE THREAD

**Issue by Gregory-Y** [deferred-impact/battledudes-issue-tracker#124]
C4 dropped by enemies could be collected (after like 2-3 seconds) and that way you could replenish lost C4 without having to waste ammo shooting it and blowing it up. Maybe you could do the same to C4 dropped by allies (instantly because there is no need to defuse), but only once they die because that way you won't steal C4 which could still be potentially useful.

Not sure if this would be QoL or gameplay.

**Comment by Haox09** [comment]
its new gameplay
but idea is fine i dont really see how this can unbalance the game

**Comment by deferred-impact** [comment]
![image](https://user-images.githubusercontent.com/87503574/218283447-9f291172-a8d0-42cc-88c0-14818bb7f4df.png)

I mentioned #77 above

**Comment by Gregory-Y** [comment]
Also I think this should only work if you have c4 equipped.
